client/blocks/importer/wordpress/import-everything/pre-migration/index.tsx

```diff
@@ -278,18 +285,11 @@ export const PreMigrationScreen: React.FunctionComponent< PreMigrationProps > =
 		);
 	}
 
-	function render() {
-		// If the source site is not capable of being migrated, we show the update info screen
-		if ( showUpdatePluginInfo ) {
-			return renderUpdatePluginInfo();
-		}
-
-		// If the target site is plan compatible, we show the pre-migration screen
-		if ( isTargetSitePlanCompatible ) {
-			return renderPreMigration();
-		}
+	function renderUpgradePlan() {
+		dispatch(
+			recordTracksEvent( 'calypso_site_migration_upgrade_plan_screen', migrationTrackingProps )
```

Comment:
Same reason as above, if we put it here it will trigger several times during the render. We might want to put it in the component.